app/api/webhook/github/route.ts

```diff
@@ -344,7 +363,7 @@ ${isComment ? `Comment by @${comment.user.login}: ${comment?.body}.` : ''}
 			octokit,
 			prisma,
 			customerId,
-			repoName: name
+			repoName: `${owner}/${name}`
```

Comment:
Looking to understand what effects this has

Reply:
Me and dexter added it so that we could inject it in the script to clone the repo in the sandbox. Since you need to recreate the https://github/owner/name.git url